### DISCUSSION TITLE
Dịch Out-of-Bag

### DISCUSSION THREAD

**Discussion by quangnhat185** [mlbvn/handson-ml2-vn#51] [Terminology] [answered]
Trong chương 7 có thuật ngữ `Out-of-Bag` chỉ các mẫu không được lấy khi sử dụng `bagging`. Mình đề nghị giữ nguyên vì từ viết tắt của nó -- `oob` được sử dụng khá nhiều trong cả văn bản và code ở chương này. 

cd: @mlbvn/handson-ml2

**Comment by minhduc0711** [comment, marked as answer]
chốt giữ nguyên là `out-of-bag` nhé
